app/views/comments/_single_comment.html.erb

```diff
@@ -14,7 +14,7 @@
     <% elsif single_comment.hidden_by_admin? && !logged_in_as_admin? %>
       <p class="message"><%= ts("(This comment is under review by an admin and is currently unavailable.)") %></p>
     <% else %>
-      <% cache single_comment do %>
+      <% cache(single_comment, :expires_in => 1.week ) do %>
```

Comment:
No this is just so changes which currently never show up such as user name changes or icon changes. Show up eventually rather than never. We have 800GB of memcached right now and effectively some things where we have no cache cleating never fall out of the cache this is just putting a maximum time for the comments to live in the cache. Any cache invalidation we do would still work it just means that if the cache invalidation is not complete ( it is not ) then there is a maximum time to live.